charts/mattermost-rtcd/values.yaml

```diff
@@ -0,0 +1,72 @@
+# Default values for mattermost-rtcd.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: angelos1989/rtcd
+  pullPolicy: Always
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "helm-test"
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+podAnnotations: {}
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+securityContext: {}
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # runAsUser: 1000
+
+service:
+  type: ClusterIP
+  # APIport is the port that websocket and API uses
+  APIport: 8045
+  # RTCport is the UDP port that RTC uses
+  RTCport: 8443
+
+ingress:
+  enabled: false
```

Comment:
I think we also need `className`.

Reply:
Thanks @stafot I have updated the whole ingress section with the newer API version